src/components/forms/DiskDeviceFormRoot.tsx

```diff
@@ -105,16 +108,29 @@ const DiskDeviceFormRoot: FC<Props> = ({
             readOnly: readOnly,
             overrideValue: formRootDevice?.pool,
             overrideForm: (
-              <StoragePoolSelector
-                project={project}
-                value={formRootDevice?.pool ?? ""}
-                setValue={(value) =>
-                  void formik.setFieldValue(`devices.${rootIndex}.pool`, value)
-                }
-                selectProps={{
-                  className: "u-no-margin--bottom",
-                }}
-              />
+              <>
+                <StoragePoolSelector
+                  project={project}
+                  value={formRootDevice?.pool ?? ""}
+                  setValue={(value) =>
+                    void formik.setFieldValue(
+                      `devices.${rootIndex}.pool`,
+                      value,
+                    )
+                  }
+                  selectProps={{
+                    className: restrictEditStoragePool
+                      ? ""
+                      : "u-no-margin--bottom",
+                    disabled: restrictEditStoragePool,
```

Comment:
sure, we still need to conditionally apply `u-no-margin--bottom` though, otherwise the help text will appear right against the selector's bottom border.

Reply:
> sure, we still need to conditionally apply `u-no-margin--bottom` though, otherwise the help text will appear right against the selector's bottom border.

Yes, that is why I kept in the suggestion above :)

Reply:
not seeing it :sweat_smile: `className` is assigned with "u-no-margin--bottom" without condition?

Reply:
I am confused, you are probably right :)